scripts/qa_crowbarsetup.sh

```diff
@@ -3256,6 +3261,38 @@ function onadmin_qa_test()
     return $ret
 }
 
+function onadmin_run_cct()
+{
+
+    addcloudrubygemrepo
+    # - install cct dependencies
+    ensure_packages_installed git-core gcc make ruby2.1-devel
+
+    # - default checkout location for mkcloud based setups:
+    mkdir -p /root/github.com/SUSE-Cloud
+    pushd /root/github.com/SUSE-Cloud/
+    git clone https://github.com/SUSE-Cloud/cct.git
+    cd cct
+    bundle install --path=.
```

Comment:
The `--path` is not needed, `.bundle/config` contains `BUNDLE_PATH: vendor/bundle` and is loaded automatically by `bundler`